src/layouts/maintenance.hbs

```diff
@@ -0,0 +1,209 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <title>{{{detag (or page.title 'Untitled')}}}{{#if site.title}} | {{{site.title}}}{{/if}}</title>
+    {{#if page.canonicalUrl}}
+    <link rel="canonical" href="{{page.canonicalUrl}}">
+    {{/if}}
+    {{#if page.description}}
+    <meta name="description" content="{{page.description}}">
+    {{/if}}
+    {{#if page.keywords}}
+    <meta name="keywords" content="{{page.keywords}}">
+    {{/if}}
+    {{#if (not (is-one-of (site-profile) 'jp' 'prod'))}}
+    <meta name="robots" content="none, noarchive">
+    {{/if}}
+    {{> head}}
+</head>
+
+<body class="flex col">
+    {{> header}}
+    {{> top-banner}}
+
+    <div class="slds-illustration slds-illustration_small">
```

Comment:
this block can be refactored into its own .hbs inside the **partials/** folder/. This makes it easier to maintain and allows us to have separate .hbs files for different languages (in case we want to have a JP version of maintenance page).